packages/shared/src/components/sidebar/MyFeedAlert.tsx

```diff
@@ -1,48 +1,106 @@
-import React, { ReactElement } from 'react';
+import React, { CSSProperties, ReactElement, ReactNode, useRef } from 'react';
+import classNames from 'classnames';
 import XIcon from '../icons/Close';
-import { Alerts } from '../../graphql/alerts';
 import { Button } from '../buttons/Button';
-import { AlertContainer, AlertCopy } from './common';
-import { Pointer, PointerColor } from '../Pointer';
+import { AlertContainer, AlertCopy, AlertWrapper } from './common';
+import Pointer, { PointerColor } from '../Pointer';
 
-const alertCopy = {
-  migrated: `Psst, your feed has a new name! We've already applied your content filters to it.`,
-  created: `🎉 Your feed is ready! Click here to manage your feed's settings`,
+interface ClassName {
+  container?: string;
+  label?: string;
+}
+
+export enum AlertPlacement {
```

Comment:
We can probably extract this into a `alertComponent` that the MyFeedAlert uses?
Let me know what you think.

Reply:
Oh, that is totally right. Didn't realize the name while refactoring. Let me work on it quickly.

Reply:
Oh, that is totally right. Didn't realize the name while refactoring. Let me work on it quickly.

What do you think of PointedAlert for the new name?

Reply:
Yeah something like `AlertPointer` will probably do.

Reply:
Awesome. Will rename the file before merging.